website/pages/blog/migrating-from-v0-to-v1.mdx

```diff
@@ -66,15 +66,13 @@ kind: destination
 spec:
   ## Required. name of the plugin
   name: "postgresql"
- 
   path: "cloudquery/postgresql"
-
+  registry: "cloudquery"
   # Required. Must be a specific version starting with v, e.g. v1.2.3
   version: "VERSION_DESTINATION_POSTGRESQL"
- 
-  ## Optional. Default: "overwrite". Available: "overwrite", "append", "overwrite-delete-stale". Not all modes are 
+  ## Optional. Default: "overwrite-delete-stale". Available:  "overwrite-delete-stale", "overwrite", "append". Not all modes are 
```

Comment:
`overwrite-delete-stale` is the current default. Not sure if this was the case when we first launched v1 (I think `overwrite-delete-stale` was always the default?)